src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
+    if (isEmpty(data)) throw new HttpException(400, "Groups don't exist");
+    
+    const groups = await this.users.update({
+      where: { id: userId },
+      data
```

Comment:
manageGroups requires validations prior to the update, it will be faster for us to validate if the user exist before using the update operation, since we will end the service with a custom throw error if one of the two entities does not exist properly.

e.g.

```
 const findUser= await this.user.findUnique({ where: { id: userId} });
 if (!findUser) throw new HttpException(409, "User doesn't exist");
```

let's follow the standard established in updateGroup